inference-engine/src/gna_plugin/frontend/weights_converter.hpp

```diff
@@ -50,15 +50,12 @@ inline bool convertWeights(InferenceEngine::CNNLayer* lp) {
     for (auto& dataItem : lp->outData) {
         dataItem->setPrecision(InferenceEngine::Precision::FP32);
     }
-    InferenceEngine::BlobMap newBlobs;
     for (auto& blob_pair : lp->blobs) {
         auto blob_name = blob_pair.first;
-        auto blob_ptr = blob_pair.second;
+        auto &blob_ptr = blob_pair.second;
         if (blob_ptr->getTensorDesc().getPrecision() == InferenceEngine::Precision::FP16) {
```

Comment:
Is this method ran only during FP16 -> FP32 conversion? Original code seems like it has no right to work since newBlobs are kinda discarded, but these changes look good for FP16 case.

Reply:
yep original code - looks partially copied from weightable layer case but without newMap reassigned back - so was completely useless, wonder why we didnt see any issues in the tests.